### ISSUE TITLE
POST to `eth/v1/builder/blinded_blocks` missing header

### ISSUE THREAD

**Issue by mcdee** [sigp/lighthouse#5405]
## Description

When receiving a POST to `/eth/v1/builder/blinded_blocks` the spec says that we should obtain the version of the blinded block in the `Eth-Consensus-Version`, however this is not set by Lighthouse.

## Version

```
lighthouse --version
Lighthouse v5.1.0-10a38a8
BLS library: blst-modern
SHA256 hardware acceleration: true
Allocator: jemalloc
Profile: maxperf
Specs: mainnet (true), minimal (false), gnosis (true)
```

## Present Behaviour

No `Eth-Consensus-Version` header is set.

## Expected Behaviour

An `Eth-Consensus-Version` header should be set as per https://ethereum.github.io/builder-specs/#/Builder/submitBlindedBlock

**Comment by michaelsproul** [comment]
How urgent is a fix here?

AFAIK we've never sent that header so this bug has existed since the builder API was defined 1.5 years ago

**Comment by michaelsproul** [comment]
Relevant section of the code is here for reference:

https://github.com/sigp/lighthouse/blob/2a3c709f8ceec4d47c1934446511660c429fe90a/beacon_node/builder_client/src/lib.rs#L140-L163

**Comment by mcdee** [comment]
It's not critical any time soon, but I do wonder how relays know the version of the block to decode without it.  I'm trying to be a bit more "by the rules"; although at current I hard-code a missing consensus version to Deneb it isn't the most future-proof of solutions.

**Comment by eserilev** [comment]
closed in #5407
